lib/src/realm_object.dart

```diff
@@ -16,37 +16,107 @@
 //
 ////////////////////////////////////////////////////////////////////////////////
 
-// ignore_for_file: native_function_body_in_non_sdk_code
+import 'native/realm_core.dart';
+import 'realm_property.dart';
 
-/**
- *  The file is intentionaly not follwoing the dart naming guidelines. The name is used from native code by convention
- */
+abstract class RealmAccessor {
+  T get<T extends Object>(RealmObject object, String name);
```

Comment:
Why not just `T get<T>(`.. anything extends Object? This is just one example.

Reply:
I did not plan on supporting null hence its written like that. But I guess I was mistaken since we need to support null values here as well.